### PR TITLE
tf ngmix

### PR DESCRIPTION
**Pull request by andrevitorelli**
Smaller PR to solve #28

### CODE REVIEW

File: autometacal/python/moments.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from .tf_ngmix.moments import get_moments
+from .tf_ngmix.gmix import create_gmix,  fwhm_to_T
+from .tf_ngmix.pixels import make_pixels
+import tensorflow as tf
+
+def gaussian_moments(images, scale, fwhm, **kwargs):
```
Comment (EiffL) [inline autometacal/python/moments.py:6]: This is the user-level API, it needs to be properly documented with docstrings. Also, the name is misleading, it's not actually returning moments but ellipticities.

**Review by EiffL — CHANGES_REQUESTED** [review]
It now runs even compiled, and everything seems to be good :-) Just flagging some needed documentation. 
To see examples our documentation style, you can check the generate_mcal_image function in metacal.py ;-)

**Comment by EiffL** [comment]
Looks good André :-) can you confirm that the test still runs without problem? I don't have a computer with me right now. If tests pass, we'll be able to merge

**Comment by andrevitorelli** [comment]
> @andrevitorelli ?

`tf_ngmix` and `metacal` tests pass here, model_fitting doesn't, it crashes.

File: autometacal/python/moments.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  
+  Args:
+    images: A bach of images Tensor
+    scale: pixel scale
+    fwhm: full width at half maximum of the gaussian filter
+    centre_x, centre_y: centre of the image, if ommited, the centre pixel of the stamp is used.
+    weights: an image containing the weights of the
+    
+  Returns:
+    Gaussian-weighted moments: e1, e2 for the batch of images. according to the a+b/()
```
Comment (EiffL) [inline autometacal/python/moments.py:18]: I think this line is not complete? It's important that the documentation of this function tells us which ellipxlticity convention we are using

Comment (andrevitorelli) [reply autometacal/python/moments.py]: @EiffL It is done, I think. I'll look into automated tests (Issue #8 ) now.

**Review by EiffL — APPROVED** [review]
LGTM

**Pull request merged**